x-pack/plugins/ml/public/components/anomalies_table/anomaly_details.js

```diff
@@ -198,28 +237,75 @@ export class AnomalyDetails extends Component {
     const anomaly = this.props.anomaly;
     const source = anomaly.source;
 
-    let anomalyDescription = `${getSeverity(anomaly.severity)} anomaly in ${anomaly.detector}`;
-    if (anomaly.entityName !== undefined) {
-      anomalyDescription += ` found for ${anomaly.entityName} ${anomaly.entityValue}`;
-    }
-
-    if ((source.partition_field_name !== undefined) &&
-        (source.partition_field_name !== anomaly.entityName)) {
-      anomalyDescription += ` detected in ${source.partition_field_name}`;
-      anomalyDescription += ` ${source.partition_field_value}`;
+    let anomalyDescription;
+    if (anomaly.entityName !== undefined (source.partition_field_name !== undefined) &&
```

Comment:
Looks like there is a boolean operator missing after the first `undefined` check, which causes the page to crash when trying to expand the row. The logic has also been changed around the checks for `anomaly.entityName` and `source.partition_field_name`. Could the previous logic be retained if possible. I tried adding `&&` after the `anomaly.entityName` and whilst that fixed the page crash, the rendered description was blank:

![image](https://user-images.githubusercontent.com/7405507/50642692-e6cec400-0f63-11e9-82bb-059ef308b182.png)

Reply:
Could you please check one more time? After adding missing boolean operator I see description value it is not blank.

Reply:
The description is still blank for all the jobs I'm looking at. For example:

This is blank:
![image](https://user-images.githubusercontent.com/7405507/50963984-103ca200-14c6-11e9-84d4-fa7e182df508.png)

Can we keep the previous logic?

Reply:
done. now we have some labels we concatenating. Checked locally, description is not blank